src/index.html

```diff
@@ -22,6 +22,404 @@
+          src="../src/images/imac.jpeg"
+          class="card_imac"
+          alt="iMac"
+        />
+        <p class="card_name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card_code">Product code: 195434</p>
+
+        <div class="card_reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card_comments">Reviews: 5</p>
+        </div>
+
+        <div class="card_price">
+          <p class="card_text">Price:</p>
+          <p class="card_dollars">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="buy"
+        >
+          buy
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          class="card_imac"
+          alt="iMac"
+        />
+        <p class="card_name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card_code">Product code: 195434</p>
+
+        <div class="card_reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card_comments">Reviews: 5</p>
+        </div>
+
+        <div class="card_price">
+          <p class="card_text">Price:</p>
+          <p class="card_dollars">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="buy"
+        >
+          buy
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          class="card_imac"
+          alt="iMac"
+        />
+        <p class="card_name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card_code">Product code: 195434</p>
+
+        <div class="card_reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card_comments">Reviews: 5</p>
+        </div>
+
+        <div class="card_price">
+          <p class="card_text">Price:</p>
+          <p class="card_dollars">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="buy"
+        >
+          buy
+        </a>
+      </div>
+      <div
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div
```

Reply:
Still not fixed.